src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private String color;
+    private Integer number;
+
+    public Ball(String color, Integer number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        String text;
+        switch (new Random().nextInt(2)) {
+            default:
+            case 0:
+                text = "You got a new ball with ";
+                break;
+            case 1:
+                text = "A strange sphere appeared before you with ";
+                break;
+            case 2:
+                text = "You have won a ball with ";
+                break;
+        }
```

Comment:
overall toString() should be consistent and return information about objects that is consistently formatted
let's remove randomizing from here?